test_util/run-all

```diff
@@ -68,6 +68,7 @@ exhibitor_zk_hosts: 192.168.65.50:2181
 exhibitor_zk_path: /zk-shared
 master_discovery: static
 check_time: 'false'
+enable_lb: 'false'
```

Comment:
@sargun Shouldn't we test both true and false? It defaults to true, but we're only testing false here?